.buildkite/autogenerate_pipeline.py

```diff
@@ -54,6 +54,7 @@
 X86_AGENT_TAGS = os.getenv('X86_LINUX_AGENT_TAGS')
 AARCH64_AGENT_TAGS = os.getenv('AARCH64_LINUX_AGENT_TAGS')
 DOCKER_PLUGIN_CONFIG = os.getenv('DOCKER_PLUGIN_CONFIG')
+TESTS_TO_SKIP = os.getenv('TESTS_TO_SKIP')
```

Comment:
I think we should add something about this in the readme as well.

Reply:
Done.